perun-core/src/main/resources/postgresChangelog.txt

```diff
@@ -6,6 +6,16 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.1.85
+alter table groups_resources drop column status, failure_case;
+create function relation_group_resource_exist(integer, integer) returns integer as 'select count(1) from groups_resources where group_id=$1 and resource_id=$2;' language sql;
+create table groups_resources_state ( group_id integer not null, resource_id integer not null, failure_cause varchar default null, status group_resource_status not null default 'PROCESSING', check ( relation_group_resource_exist(group_id, resource_id) != 0 ));
```

Comment:
We should also insert existing statuses and failure causes from `groups_resources` to `groups_resources_state`. Therefore, creating table `groups_resources_state` and inserting values into it should be executed before dropping the columns in `groups_resources`.

Reply:
Insert added and drop moved, thanks.